### DISCUSSION TITLE
Reuse login fields / login form (login.blade.php) elsewhere

### DISCUSSION THREAD

**Discussion by martingrondein-nls** [filamentphp/filament#3573] [Help]
Hi there

I'd like to reuse the `/admin/login` login form found in the `login.blade.php` file else where on another Blade template.  Any ideas on how best to approach this.

Simply copying and pasting this doesn't work:
```php
<form wire:submit.prevent="authenticate" class="space-y-8">
    {{ $this->form }}

    <x-filament::button type="submit" form="authenticate" class="w-full">
        {{ __('filament::login.buttons.submit.label') }}
    </x-filament::button>
</form>
```

Would like to have it still redirect and log me in to the Filament Dashboard upon logging in.

**Comment by danharrin** [comment]
It will need to be a Livewire component, using our form builder, to render the form. Check out the Login class

**Reply by martingrondein-nls** [reply]
Thanks @danharrin  - I've copied the package's `vendor/filament/filament/src/Http/Livewire/Auth/Login.php` and popped it in my app's `app/Filament/Auth/Login.php` location while also changing the namespace to just `App\Filament\Auth` as to not conflict with the package.  

I've then updated my apps' Filament config to point to my "new" class:

```php

use App\Filament\Auth\Login;

return [
    ...
    'auth' => [
        'pages' => [
            'login' => Login::class,
        ],
    ],
];
```

But still struggling to get the Livewire component to render
